### PR TITLE
Add deprecation notice to README.md

### PR DESCRIPTION
**Pull request by eddiezane**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# DEPRECATION NOTICE
+
+This library is being deprecated in favor of the [Clarifai Apple SDK](https://github.com/Clarifai/clarifai-apple-sdk).
```
Comment (dcirne) [inline README.md]: `is being` or `has been`?

Comment (abhishek-mathur) [inline README.md]: IMO, should be `is being`. Check with Kunal too?

Comment (dcirne) [inline README.md]: Ok. We can use `is being` now and change to `has been` later down the road.

Comment (kunal732) [inline README.md]: `is being`.

**Pull request merged**